module-2/backend/week_one.md

```diff
@@ -5,30 +5,79 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+GET - retrieves information from the client
+POST - takes information from the app and puts it into the view

```

Comment:
POST generally creates a new resource in our database. One of the results is that we end up being able to access it in the view, but that's not the purpose of a POST request.